src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -44,20 +44,25 @@ const CategoryDropdown = ({
   const dispatch = useAppDispatch()
   const { openModal, closeModal } = useModalContext()
 
-  const handleResponseError = (error?: ErrorResponse) => {
-    dispatch(
-      openAlert({
-        severity: snackbarVariants.error,
-        message: getErrorKey(error)
-      })
-    )
-  }
+  const handleResponseError = useCallback(
+    (error?: ErrorResponse) => {
+      dispatch(
+        openAlert({
+          severity: snackbarVariants.error,
+          message: getErrorKey(error)
+        })
+      )
+    },
+    [dispatch]
+  )
 
-  const { response: allCategoriesNames, fetchData: fetchAllCategoriesNames } =
-    useAxios<CategoryNameInterface[]>({
-      service: ResourceService.getResourcesCategoriesNames,
-      defaultResponse: [],
-      fetchOnMount: true
+  const { data: allCategoriesNames, refetch: fetchAllCategoriesNames } =
```

Comment:
Don't use refetch manually, use an ability with query invalidation inside `useMutation`
```suggestion
  const { data: allCategoriesNames } =
```